README.md

```diff
@@ -148,20 +223,34 @@ test('sometimes async gets rejected', function(assert){
 
 - `description`: (String) optional - The description of the module
 
-- `callbacks`: (Object) optional - Normal QUnit callbacks (setup and
-  teardown), with addition to `needs`, which allows you specify the
-  other units the tests will need.
+- `callbacks`: (Object) optional
+   - QUnit callbacks (`beforeEach` and `afterEach`)
```

Comment:
Nope, `beforeEach` and `afterEach` are the newer QUnit 2.0 style method names. `setup` and `teardown` are deprecated in QUnit itself, so we should suggest using using the new style.  

FWIW - ember-watson has a really nice command to fix this automatically for you.